db/db_test.go

```diff
@@ -8,7 +8,7 @@ import (
 func SetupDatabaseForTests(t *testing.T) {
 	SetupTestDatabase()
 	tables := ShowTables()
-	assert.Equal(t, 3, len(tables))
+	assert.Equal(t, 4, len(tables))
```

Comment:
It might be better to change this to assert that there is at least one table in the database, or change this entirely so it's not dependent on number of tables but instead checking that the schema is set up.

Or, it's not even needed entirely. Can be done in followup PR

Reply:
Good idea, although it is nice to check that all tables are setup correctly as the script could encounter an error after creating a few tables.

Reply:
I mean the tests will fail and mysql will throw the error saying that the table isn't created. So you'll know this information anyways if this happens.

Reply:
Currently the error isn't caught since go isn't directly executing these commands, but instead they are being executed via shell script. So a table can fail and we would not currently know.

Reply:
I see. In a follow up PR, we could get rid of `db_test.sh` and `db_test_setup.sh` entirely and just do everything with the `mysql -e` syntax. A lot cleaner, in my opinion, and we would see failures directly from go then.